src/module-elasticsuite-virtual-category/Plugin/Catalog/Category/SaveProductsPositions.php

```diff
@@ -101,16 +101,36 @@ function () use ($category) {
      */
     private function getAffectedProductIds($category)
```

Comment:
The fix on getAffectedProductIds() will only apply for "index on save" behavior, though